src/Compilers/CSharp/Test/Emit3/Semantics/ExperimentalAttributeTests.cs

```diff
@@ -29,6 +29,8 @@ public sealed class ExperimentalAttribute : Attribute
         public ExperimentalAttribute(string diagnosticId) { }
 
         public string? UrlFormat { get; set; }
+
+        public string? Message { get; set; }
```

Comment:
How should we track the update to the [spec](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-12.0/experimental-attribute.md)?
One option would be to move that doc to roslyn/docs/features and update it there, as this is more a compiler feature than a language feature (we enforce regardless of language version) anyways.

Reply:
I can update the spec in place. I see no value in moving the document around for the API change.